waict-signalling.md

```diff
@@ -1,55 +1,135 @@
-# WAICT Transparency Signaling
+# WAICT Enforcement Signaling
 
 # Introduction
 
-This spec describes the ways a client signals to a server that they support WAICT, and vice-versa.
+This spec describes the ways a server signals to a client to enforce the use of WAICT.
+
+# Conventions
+
+This document uses Structured Field Values for HTTP (RFC 8941).
 
 # Client-side Signalling
 
-Clients MAY signal that they support transparency. Doing so will allow the server to avoid sending unnecessary transparency information to the client. To signal transparency support, the client includes the request header `WAICT-Transparency-Supported: 1` in their requests to the site. Future versions of this specification may define different version numbers.
+Clients SHOULD signal that they support WAICT. Doing so will allow the server to avoid sending unnecessary information to the client.
```

Comment:
Suggest for clarity:

```suggestion
Clients that support WAICT SHOULD signal the server, to allow the server to avoid sending unnecessary information to the client.
```